core/converters.go

```diff
@@ -90,6 +90,20 @@ func GetShardIDString(shardID uint32) string {
 	return fmt.Sprintf("%d", shardID)
 }
 
+// ConvertShardIDToUint32 converts shard id from string to uint32
+func ConvertShardIDToUint32(shardIDStr string) (uint32, error) {
+	if shardIDStr == "metachain" {
+		return MetachainShardId, nil
+	}
+
+	shardID, err := strconv.ParseInt(shardIDStr, 10, 64)
```

Comment:
`ParseInt(shardIDStr, 10, 32)` - but it really does not matter in this case, actually.

Reply:
it was copy paste from existing code.